samples/ChangefeedMigrationSample/Program.cs

```diff
@@ -0,0 +1,222 @@
+        {
+            Console.WriteLine("Worker opened, {0}", context.PartitionKeyRangeId);
+            return Task.CompletedTask;  // Requires targeting .NET 4.6+.
+        }
+
+        public Task CloseAsync(ChangeFeedObserverContext context, ChangeFeedObserverCloseReason reason)
+        {
+            Console.WriteLine("Worker closed, {0}", context.PartitionKeyRangeId);
+            return Task.CompletedTask;
+        }
+
+        public Task ProcessChangesAsync(ChangeFeedObserverContext context, IReadOnlyList<Document> docs)
+        {
+            DocumentCollectionInfo destCollectionLocation = new DocumentCollectionInfo
+            {
+                Uri = new Uri("httpsURI"),
+                MasterKey = "PrimaryKey",
+                DatabaseName = "DestDBname",
+                CollectionName = "DestCollName"
+            };
+
+            DocumentClient newClient = new DocumentClient(destCollectionLocation.Uri, destCollectionLocation.MasterKey);
+
+            newClient.CreateDatabaseIfNotExistsAsync(new Database { Id = destCollectionLocation.DatabaseName });
+
+            // create dest collection if it does not exist
+            DocumentCollection destCollection = new DocumentCollection();
+            destCollection.Id = destCollectionLocation.CollectionName;
+
+            //destCollection.PartitionKey.Paths.Add("add partition key if applicable");
+
+            newClient.CreateDocumentCollectionIfNotExistsAsync(
+                UriFactory.CreateDatabaseUri(destCollectionLocation.DatabaseName),
+                destCollection,
+                new RequestOptions { OfferThroughput = 500 });
+
+            Console.WriteLine("Change feed: total {0} doc(s)", Interlocked.Add(ref s_totalDocs, docs.Count));
+            foreach (Document doc in docs)
+            {
+                Console.WriteLine(doc.ToString());
+                newClient.UpsertDocumentAsync(
+                    UriFactory.CreateDocumentCollectionUri(destCollectionLocation.DatabaseName, destCollectionLocation.CollectionName),
+                    doc);
+            }
+            return Task.CompletedTask;
+        }
+    }
+
+    class Program
+    {
+        // Modify EndPointUrl and PrimaryKey to connect to your own subscription
+        private string uri = "https://URI";
+        private string secretKey = "primaryKey";
+
+        private string dbName = "yourdbName";
+        private string monitoredCollectionName = "monitoredCollectionName";
+        private string leaseCollectionName = "leaseCollectionName";
+
+        static void Main(string[] args)
+        {
+            Console.WriteLine("ChangeFeed App");
+            Program newApp = new ChangefeedMigrationSample.Program();
+            // Thread 1 comment out for thread 2 
+            newApp.RunChangeFeedProcessor();
+            // Thread 2 comment out for thread 1 
+            // UpdateDb(EndPointUrl, PrimaryKey); 
+            Console.ReadKey();
```

Comment:
Console.WriteLine("Running... Press any key to stop.");
Probably would be better to press Enter to stop rather than any key, so it becomes Console.ReadLine().